### ISSUE TITLE
Incorporate changes in the beancount-"booking"-branch into fava

### ISSUE THREAD

**Issue by aumayr** [beancount/fava#289]
@blais on the beancount mailing list:

> Big progress on the long awaited 'booking' branch today, I'm really excited about this:
> I managed to implement automatic, working FIFO and LIFO booking methods on my gigantic personal file!
> This means that for the very first time I'm able to process my personal ledger entirely with the new code, and so I'll begin using that exclusively.
> 
> I'm not ready for unveiling yet, but this is the culmination of a fair amount of changes.
> The 'booking' branch will bring many features useful for investing:
> - The original acquisition dates will be kept on all lots (already working)
> - Many lot reduction methods will be supported: strict identification (like now), FIFO, LIFO, average cost, etc. (done except for average cost).
> - It will be possible to easily extract and report all trades carried out within a period.
> - The interpolation will be significantly more powerful (also already done).
> 
> I know a few of you are waiting on the average cost booking.
> It shouldn't be too hard to implement it on top.
> I'll work on that very next.

I have not figured out yet what this actually means in changes to fava, and honestly do not have too much experience with using beancount for tracking investments, but I want to start this issue as a discussing point for incorporating these beancount-changes into fava, displaying new information that is available appropriately, etc.

So if you have any ideas how to adapt fava with new charts, tables, etc., feel free to comment! :-)

**Comment by blais** [comment]
Here are some ideas:
- The Holdings page should include the acquisition date of each lot
- We can now create a new page to list all the trades closed during the reporting period (and their individual P/L)
- In the trial balance / update dates page, it may be worthwhile to list the booking method associated with each account (using the default value if left unset)
- Because the input syntax is more lax, it may be useful to provide the output of bean-doctor context in a tooltip instead of having to navigate to a dedicated page.

**Comment by pfrancois** [comment]
for my point of view , it's almost work out of the box.
the only problem i see now is that the market value is wrong. it show the historical cost value and it should show the market value.

**Comment by aumayr** [comment]
@yagebu I do not really know what changes/issues/tasks should be created because of this. Do you have any insight if we should actually change anything or if we can close this?

**Comment by yagebu** [comment]
This mainly:

> We can now create a new page to list all the trades closed during the reporting period (and their individual P/L)

I'll open an issue for that soon and close this one here then.

**Comment by yagebu** [comment]
Now tracked in #715
